Adding first set of compliance content. (#27247)
demisto/content: Packs/XSIAMCompliance_PCI_DSS/README.md

## **Generate Compliance Dashboards and Reports**

> The PCI DSS compliance pack provides a dashboard and report template to ease your compliance monitoring and tracking. You can also use it as evidences in the yearly compliance audits and show auditors how data is tracked in XSIAM.

Reviewer: ```suggestion
The PCI DSS compliance pack provides a comprehensive dashboard and report template to easily monitor your Cortex XSIAM data compliance regulations and provide evidence for compliance audits. The dashboard and report aggregate and reference all data mapped to the Cortex Data Model (XDM).
```

> The dashboard and report in this pack utilizes XQL queries that reference the XSIAM data model (XDM). All data that has been onboarded to XSIAM and mapped to the data model will be automatically included in this dashboard and report.

Reviewer: ```suggestion

```

**Note:** This pack is currently offered as a free beta of the Compliance module for a limited time. In the future, this module will be sold separately on top of XSIAM.

## **What does this pack do?**
- Install dashboard and report templates for the PCI DSS regulations.

> - Reference data that has been mapped to the data model.

Reviewer: ```suggestion
- Reference data that has been mapped to the Cortex Data Model (XDM).
```

## **Generate Compliance Dashboards and Reports**

> The compliance pack provides dashboards and reports for a variety of regulations. These dashboards and reports utilize queries which reference the XSIAM data model. Once data has been onboarded to XSIAM and datasets have been mapped to the data model, these dashboards and reports will query the data model to generate the dashboards and reports. Use these dashboards and reports as part of yearly compliance audits to show auditors how data is being tracked in XSIAM.

Reviewer: ```suggestion
The compliance pack provides dashboards and reports for a variety of regulations. These dashboards and reports utilize queries which reference the Cortex XSIAM data model. Once data has been onboarded to Cortex XSIAM and datasets have been mapped to the data model, these dashboards and reports will query the data model to generate the dashboards and reports. Use these dashboards and reports as part of yearly compliance audits to show auditors how data is being tracked in Cortex XSIAM.
```

> The compliance pack provides dashboards and reports for a variety of regulations. These dashboards and reports utilize queries which reference the Cortex XSIAM data model. Once data has been onboarded to Cortex XSIAM and datasets have been mapped to the data model, these dashboards and reports will query the data model to generate the dashboards and reports. Use these dashboards and reports as part of yearly compliance audits to show auditors how data is being tracked in Cortex XSIAM.

Reviewer: ```suggestion
The PCI DSS compliance pack provides a dashboard and report template to ease your compliance monitoring and tracking. You can also use it as evidences in the yearly compliance audits and show auditors how data is tracked in XSIAM.
The dashboard and report in this pack utilizes XQL queries that reference the XSIAM data model (XDM). All data that has been onboarded to XSIAM and mapped to the data model will be automatically included in this dashboard and report.
```

> **Note:** This pack is currently offered as a free beta for the Compliance module. However, in the future, this module will be sold separately.

Reviewer: ```suggestion
**Note:** This pack is currently offered as a free beta of the Compliance module for a limited time. In the future, this module will be sold separately on top of XSIAM.
```

## **What does this pack do?**

> -Install dashboard and report templates for the following regulations: PCI DSS

Reviewer: ```suggestion
- Install dashboard and report templates for the PCI DSS regulations.
```

> -Reference data that has been mapped to the data model.

Reviewer: ```suggestion
- Reference data that has been mapped to the data model.
```

> -Produce evidence for audits/regulations.

Reviewer: ```suggestion
- Produce regulation evidence for audits.
```
